spec/model/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context 'validation content' do
+    it 'ensures content presence' do
+      comment = Comment.new(content: '').save
+      expect(comment).to eql(false)
+    end
+
+    it 'should save successfully' do
+      comment = Comment.new(content: "We've got something rolling").save
+      expect(comment).to eql(false)
+    end
```

Comment:
The test statement is confusing. You are saying the comment "should save successfully" but you are testing the falseness i.e., comments "should not be saved". Please fix this.